infrastructure/cdn-in-a-box/optional/vpn/README.rst

```diff
@@ -39,7 +39,7 @@ On Linux, you could choose ``openvpn``. Take ubuntu/debian as an example, you ca
 
 .. code-block:: shell
 
-    apt-get update && apt-get install -y openvpn
+    apt-get update && apt-get --no-install-recommends install -y openvpn
```

Comment:
Same as above

Reply:
I general updated "apt-get install" with "apt-get --no-install-recommends install" as a best practice for apt-get wherever applicable  , doesn't have any issue and didn't looked for specific use case

Reply:
Agree with @ocket8888—We're not in the business of policing a user's own APT configuration.